packages/storybook/stories/VideoGallery/VideoGallery.stories.tsx

```diff
@@ -23,22 +27,52 @@ const getDocs: () => JSX.Element = () => {
     <>
       <Title>VideoGallery</Title>
       <Description>
-        VideoGallery represents a layout of video tiles for a specific call. It displays a
-        [VideoTile](./?path=/docs/ui-components-videotile--video-tile) for the local user as well as for each remote
-        participant who has joined the call.
+        VideoGallery lays out the local user and each remote participant in a call in a
+        [VideoTile](./?path=/docs/ui-components-videotile--video-tile) component. The VideoGallery componet is comprised
```

Comment:
componet => component

Reply:
The VideoGallery componet is comprised ofa Grid Layout, Horizontal Gallery, and the Floating Local Video.

The VideoGallery component is made up of a [Grid Layout](./?path=), [Horizontal Gallery](./?path=), and a [Local Video Tile](./?path=).